corehq/apps/case_search/views.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from corehq.apps.domain.decorators import require_superuser
+from corehq.apps.domain.views import DomainViewMixin
+from dimagi.utils.web import json_response
+from django.views.generic import TemplateView
+
+
+# Create your views here.
+class CaseSearchView(DomainViewMixin, TemplateView):
+    template_name = 'case_search/case_search.html'
+    urlname = 'case_search'
+
+    @require_superuser
+    def get(self, request, *args, **kwargs):
+        return self.render_to_response(self.get_context_data())
+
+    @require_superuser
+    def post(self, request, *args, **kwargs):
+        from corehq.apps.es.case_search import CaseSearchES
+        query = json.loads(request.POST.get('q'))
+        case_type = query.get('type')
+        search_params = query.get('parameters')
```

Comment:
Yeah. This is actually a debug view which actually allows you to search without parameters (i.e. it just searches all cases in the domain).